lib/GenomeFileUtil/core/GenbankToGenome.py

```diff
@@ -356,11 +357,22 @@ def _save_assembly(self, genbank_file, params):
             if "KBaseGenomeAnnotations.Assembly" not in ret['info'][2]:
                 raise ValueError("{} is not a reference to an assembly"
                                  .format(assembly_ref))
-            assembly_contig_ids = set(ret['data']['contigs'].keys())
-            unmatched_ids = set(self.contig_seq.keys()) - assembly_contig_ids
-            if unmatched_ids:
-                raise ValueError("The genbank file contains contigs which are not present in the "
-                                 "supplied assembly".format(", ".join(unmatched_ids)))
+            unmatched_ids = list()
+            unmatched_ids_md5s = list()
+            print "SELF CONTIG SEQ KEYS: " + str(self.contig_seq.keys())
+            print "SELF CONTIG SEQ : " + str(self.contig_seq)
+            print "PREV UPLOADED CONTIGS : " + str(ret['data']['contigs'])
```

Comment:
I think we can drop these print statements

Reply:
Yeah my Bad.  Forgot cleanup

Reply:
Ok done. Do I need to do a new PR?